src/atoms/LinkWrapper/link_wrapper.module.scss

```diff
@@ -1,6 +1,7 @@
 .link {
   color: color('neutral-4');
   display: inline-block;
+  text-decoration: underline;
```

Comment:
@jmcolella the `text-decoration` for the anchor tags is currently coming from the agent user stylesheet, which only adds it if there's a `href` attribute on the tag. I think we are going to end having a lot of LinkWrappers that only take `onClick` which is why I added it, although there might be a better/different solution?